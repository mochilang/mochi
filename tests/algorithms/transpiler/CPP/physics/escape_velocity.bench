{
  "duration_us": 554,
  "memory_bytes": 3989504,
  "name": "main"
}