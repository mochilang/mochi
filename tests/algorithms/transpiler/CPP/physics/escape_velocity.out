11185.977999999999
2376.3069999999998
60199.544999999998